tests/std/tests/P2387R3_bind_back/test.cpp

```diff
@@ -101,6 +99,33 @@ constexpr bool test_constexpr() {
     assert(move(bound5)() == "modifiable rvalue");
     assert(move(as_const(bound5))() == "const rvalue");
 
+    // Test PMFs.
+    Cat cat{"Peppermint"};
+    auto bound2 = bind_back(&Cat::noise, "meow");
+    assert(bound2(cat) == "Peppermint says meow"); // // call with reference
+    auto bound3 = bind_back(&Cat::noise, "MEOW");
+    cat.name    = "Fluffy";
+    assert(bound3(&cat) == "Fluffy says MEOW"); // call with pointer

```

Comment:
We could additionally test calling a bound PMF with a `reference_wrapper` (similar to the `bind_front` coverage, except that here the `reference_wrapper` would be passed at the final invocation). Unlikely to find any bugs (as the existing coverage is sufficient to verify that we're using the `invoke` protocol), but good for consistency.